cwf/k8s/cwf_operator/docker/Dockerfile

```diff
@@ -20,9 +20,12 @@ RUN apt-get update && apt-get install -y bzr curl daemontools gcc
 
 # Install Golang 1.18
 WORKDIR /usr/local
-RUN curl https://artifactory.magmacore.org/artifactory/generic/go1.18.linux-amd64.tar.gz --remote-name --location && \
-    tar -xzf go1.18.linux-amd64.tar.gz && \
-    cp -r go/bin/* /usr/local/bin/
+ARG GOLANG_VERSION="1.18"
+RUN GO_TARBALL="go${GOLANG_VERSION}.linux-amd64.tar.gz" \
+ && curl https://artifactory.magmacore.org/artifactory/generic/${GO_TARBALL} --remote-name --location \
+ && tar -xzf ${GO_TARBALL} \
+ && cp -r go/bin/* /usr/local/bin/ \
```

Comment:
If there is more than the go binary in the archive, we will have unwanted files in the image this way. If we would only extract the needed files, this could be avoided.
https://github.com/magma/magma/blob/a68bf5d633b8167cc5b96196e2dd0cdbfa2841c6/orc8r/cloud/deploy/orc8r_deployer/docker/Dockerfile#L48-L53

Reply:
This actually also holds for the above. However, we could also avoid the separate copy-step here, by making use of all tar functionality.

Reply:
While I do not know, how big the package is, it is than usually a good idea to only sym-link the required binary in the `/usr/local/bin` directory than, imho.